.github/dependabot.yml

```diff
@@ -0,0 +1,15 @@
+version: 2
+updates:
```

Comment:
I did not add the `docker` `package-ecosystem` because our `Dockerfile` `FROM` instructions currently contain template variables.